extensions/blocks/anchor-fm/templates/index.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * WordPress dependencies
+ */
+import { createBlocksFromInnerBlocksTemplate, } from '@wordpress/blocks';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { getIconColor } from '../../../shared/block-icons';
+
+// Templates.
+
+function spotifyTemplate( { spotifyShowUrl, spotifyImageUrl } ) {
```

Comment:
Now that we have that full block structure, I expected this to appear right below the podcast image, in the left column, but maybe that's just me. :)

Reply:
Something to consider. The template structure can change in the near future. For now, we focus on its functionality. Here there details about this [episode template](361-gh-Automattic/dotcom-manage), just FYI.
Thanks.